src/dynamo/dynamo/outputplugins/tickerproperty/radialdist.cpp

```diff
@@ -111,6 +120,9 @@ namespace dynamo {
   
     for (const shared_ptr<Species>& sp1 : Sim->species)
       for (const shared_ptr<Species>& sp2 : Sim->species)
+        {
+         if (sp1->getName() == std::get<0>(rdfpairs) && sp2->getName() == std::get<1>(rdfpairs))
```

Comment:
Two comments:
First, I'd like to preserve the default behaviour of collect everything, as that's most people's use-case. Maybe you have a species index to ignore rather than a species index to sample?

Second, string comparison is "slow", so I'd prefer to check something like species index?

Reply:
I think this should be doable. My initial goal was to have the abilities to specify a list of pairs in the command line, or all, and it would do all pairs requested. We could then make all the default if no command-line argument is given. I'll try to figure that part out, and then work on using Index for comparison rather than Name.

Reply:
Great!